server/src/main/java/com/netflix/conductor/server/ConductorServer.java

```diff
@@ -156,10 +156,10 @@ public HostToken getTokenForHost(Host host, Set<Host> activeHosts) {
 			jedis = new JedisMock();
 			try {
 				EmbeddedElasticSearch.start();
-				if(System.getProperty("workflow.elasticsearch.url") == null) {
+				if(cc.getProperty("workflow.elasticsearch.url", null) == null) {
```

Comment:
Making sure that we're going through ConductorConfig to get configuration values so that environment variables are considered.